docs/conf.py

```diff
@@ -30,6 +30,7 @@
 
 extensions = [
     "sphinx.ext.autodoc",
+    "sphinx.ext.autosummary",
```

Comment:
I don't see where is the `.. autosummary::` used in the docs. Is this extension used anywhere?

Reply:
You are correct. I probably thought it was required for `automodule` at the time.